src/ApplicationInsights.ServiceFabric.Native/Net45/DependencyTrackingModule/RequestResponseHeaders.cs

```diff
@@ -0,0 +1,47 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.ApplicationInsights.ServiceFabric.Module
+{
+    internal class RequestResponseHeaders
```

Comment:
header names are already defined in `ServiceRemotingLoggingStrings.cs`
could you merge this file with ServiceRemotingLoggingStrings?